README.md

```diff
@@ -132,6 +132,98 @@ In order to support atomify turtles all the way down, you can also specify your
 
 For detailed information on configuring Rework plugins in package.json see the [relevant section](https://github.com/Techwraith/atomify-css#packagejson-config) of the atomify-css README.
 
+## Mini Tutorial
+### How to build a re-usable component
+(__Warning:__ _The following example is only interesting from a technical and not at all from an esthetical perspective)_
```

Comment:
On npm, yea, we prefix everything with `atomify-`, but perhaps the ecosystem should standardize on using a keyword `atomify-component`.

On CSS namespaces, we've got a styleguide that's still kinda a work in progress, but should answer your questions: https://github.com/Getable/styleguide-css